BOOKBBANG/BOOKBBANG/Views/Book/AddBook/AddBookFinal.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AddBookFinal.swift
+//  BOOKBBANG
+//
+//  Created by 이연정 on 8/1/24.
+//
+
+import SwiftUI
+
+struct AddBookFinal: View {
+  var body: some View {
+    VStack{
+      Text("빵 굽기를 완료했습니다")
+        .font(.system(size: 24, weight:.bold))
+        .foregroundStyle(.typo100)
+        .padding(.bottom, 5)
+        .frame(maxWidth: .infinity, alignment: .leading)
+        .padding(.leading, 22)
+      
+      Text("축하합니다. 구워진 빵을 리스트에서 확인해볼까요?")
+        .font(.system(size: 12))
+        .foregroundStyle(.typo50)
+        .frame(maxWidth: .infinity, alignment: .leading)
+        .padding(.leading, 22)
+      
+      // 오븐 이미지 아님, 다다가 그릴 예정
+      Image("oven")
```

Comment:
```suggestion
      Image(.oven)
```

하드코딩 지양하기. 그럼 하드코딩이란 무엇일까용?

Reply:
구리스 무슨말이세요?

Reply:
하드코딩이란?
데이터를 코드 내부에 직접 입력하는 것
외부 입력에 대한 유연성이 없고 확장성이 낮은 코딩

저의 코드는 100프로 하드코딩이였군요...
구리스 덕에 조금이나마 덜어내보겠습니다